go_attack_utils/src/sgf_parser/game_info.py

```diff
@@ -33,7 +33,9 @@ def find_sgf_files(
     sgf_paths = []
     directories_scanned = 0
     for dirpath, _, filenames in os.walk(root):
-        sgf_filenames = [x for x in filenames if x.endswith(".sgfs")]
+        sgf_filenames = [
+            x for x in filenames if x.endswith(".sgfs") or x.endswith(".sgf")
```

Comment:
ran into an issue where originally I was copying over SGFs with the suffix `".sgf"`, but then they were getting missed by `find_sgf_files()` and weren't showing up in `game_infos.csv`